controllers/AuthController.js

```diff
@@ -0,0 +1,62 @@
+const BaseController = require('./BaseController');
```

Comment:
Please setup eslint and fix codding style. You can use same configuration witch we are using in kr-api

Reply:
done